boxoffice/mailclient.py

```diff
@@ -30,3 +31,18 @@ def send_participant_assignment_mail(order_id, subject="Please tell us who's com
         msg.html = html
         msg.body = html2text(html)
         mail.send(msg)
+
+
+def send_line_item_cancellation_mail(line_item_id, subject="Ticket Cancellation"):
```

Comment:
As this is a background job, use the `@job` decorator. The `app.test_request_context()` structure should only be used at an entry point, which this function is, but it's hard to tell without the decorator.